conf/contributors.xml

```diff
@@ -2601,11 +2601,13 @@ Please keep the list in alphabetical order by last name!
  location="123, avenue Albert Thomas, 87060 Limoges Cedex, France"
  description="GAP interface bug fix"
  url="http://www.unilim.fr/pages_perso/matthias.meulien/index.en.html"
- trac="mmeulien"/>
+ trac="mmeulien"
+ github="orontee"/>
```

Comment:
How about merging these two into one?

Reply:
You're right. I added a commit for this.